neoload/README.md

```diff
@@ -0,0 +1,43 @@
+# NeoLoad integration
+
+## Overview
+
+[Tricentis NeoLoad][1] simplifies and scales performance testing for APIs and microservices, as well as end-to-end application testing through protocol and browser-based capabilities.
+
+With the NeoLoad integration, you can track performance metrics of NeoLoad tests to:
+
+- Correlate application performance with load testing metrics.
+- Analyze and visualize NeoLoad metrics using the out-of-the-box Dashboard or [Metrics Explorer][7].
+
+## Setup
+
+### Configuration
+
+For detailed instructions on NeoLoad configuration, follow the [NeoLoad documentation][2]. Choose which metrics to send in the "Push Counters" configuration of the Datadog Connector in NeoLoad. Available since NeoLoad 9.1.
+
+When Datadog receives `NeoLoad.*` metrics, the NeoLoad integration tile is installed automatically, and the default NeoLoad dashboard is added to your dashboard list.
+
+
+## Data Collected
+
+### Metrics
+
+See [metadata.csv][3] for a list of metrics provided by this integration.
+
+### Events
+
+All NeoLoad performance tests events are sent to your [Datadog Events Explorer][4].
+NeoLoad sends events to the Datadog API when a performance test starts and ends.
+Set the option in the "Push Counters" configuration of the Datadog Connector in NeoLoad. Available since NeoLoad 9.1.
```

Comment:
Is this setting for sending events to Datadog? Can you clarify what this setting does?

Reply:
Yes this setting enable/disable sending events to Datadog. It is enabled by default, and it is pretty obvious for the user lookig at NeoLoad UI.